docs/body.md

```diff
@@ -494,6 +494,7 @@ event log (e.g. "System") that you are interested in, and in the
 EventQuery enter the filter for events. The filter can be either
 XPath XML taken from a Windows Event Viewer Custom View or a PowerShell Where-Object block.
 The max age field is only used the first time the datasource successfully runs. Subsequent runs will only pull events from the last successful polling cycle.
+Also users can apply their own transform to the events.
```

Comment:
remove and add the following to line 555, just before "#### Powershell Examples"

Note: Collection errors are sent with the WindowsEventLogCollection event class key.
Use an event class mapping with a transform to forward the event to a specific
event class. These include connection and other Powershell issues.